source/_integrations/nut.markdown

```
@@ -97,6 +97,186 @@ The following switches are available for each switchable outlet:
 The integration uses {% term polling %} to retrieve data from the NUT
 server. The default polling interval is once every 60 seconds.
 
+## Supported functionality
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)